lib/ansible/plugins/action/__init__.py

```diff
@@ -1033,3 +1024,33 @@ def _find_needle(self, dirname, needle):
 
         # if missing it will return a file not found exception
         return self._loader.path_dwim_relative_stack(path_stack, dirname, needle)
+
+    def _get_admin_users(self):
+        '''
+        Returns a list of admin users that are configured for the current shell
+        plugin
+        '''
+        # fallback for old custom plugins w/o get_option
+        try:
+            admin_users = self._connection._shell.get_option('admin_users')
+        except AnsibleError:
+            admin_users = ['root']
+        return admin_users
+
+    def _deescalated_user(self):
+        '''
+        The user is not the same as the connection user and is not part of the
+        shell configured admin users
+        '''
+        # if we don't use become then we know the user is deescalated
```

Comment:
I think the comment here should be the opposite:
```
# If we don't use become then we know we aren't switching to a different unprivileged user
```

Reply:
yep, will fix up.